src/media/css/app-list-filters.styl

```diff
@@ -90,11 +90,18 @@ App listing filters.
     display: none;
 }
 
-.app-list-filters + .app-list {
+body:not([data-mkt-nav--enabled]) .app-list-filters + .app-list {
```

Comment:
That is better, although we're going to wipe out the flag soon and this might the only case of offense so meh.